IFC4x3/Properties/c/CoefficientOfLinearExpansion_1f_$rsY_rFM9ETgqpq1yXr/Documentation.md

```diff
@@ -0,0 +1 @@
+Quantity characterizing the variation with thermodynamic temperature T of the distance l between two points of a body, under given conditions (IEC 113-04-27). The ratio is defined per Kelvin.
```

Comment:
If reuse, I would propose modifying the description of `ThermalExpansionCoefficient`.